LayoutTests/css-typedom/attributeStyleMap.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html><!-- webkit-test-runner [ CSSTypedOMEnabled=true ] -->
+<!DOCTYPE html><!-- webkit-test-runner -->
```

Comment:
You can remove `<!-- webkit-test-runner -->` entirely here and in other tests that were only setting this pref.